apogee/tools/read.py

```diff
@@ -1337,3 +1394,6 @@ def _warn_astroNN_distances():
 
 def _warn_astroNN_ages():
     warnings.warn("Adding ages from Mackereth, Bovy, Leung, et al. (2019)")
+
+def _warn_astroNN_orbits():
+    warnings.warn("Adding orbits and Galactocentric coordinates from DR16 astroNN VAC")
```

Comment:
Maybe ref. galpy papers here (general and yours)?

Reply:
done!